explore-assistant-cloud-run/cloudrun_build.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+# Load environment variables from the .env file
+# Make sure .env is in the same directory as this script
+if [ -f .env ]; then
+  # Automatically export all variables defined in .env
+  set -a
+  source .env
+  set +a
+else
+  echo ".env file not found!"
+  exit 1
+fi
+
+# Configuration
+
+## env_var. Refer to ./.env
+REGION_NAME="$REGION_NAME"
+PROJECT_ID="$PROJECT_NAME"
+IMAGE_NAME="$IMAGE_NAME"
+
+## hard coded
+REPO_NAME="looker-explore-assistant"
+REPOSITORY_REGION="$REGION_NAME-docker.pkg.dev"
+TAG="latest"
+IMAGE="$REPOSITORY_REGION/$PROJECT_ID/$REPO_NAME/$IMAGE_NAME:$TAG"
+
+echo "Building Docker image..."
+docker build -t $IMAGE .
+
+# echo "Testing Docker image locally..."
+# docker run -p 8080:8080 gcr.io/$PROJECT_ID/$IMAGE_NAME:$TAG
+
+echo "Pushing Docker image to GCR..."
+gcloud auth configure-docker $REPOSITORY_REGION
+
+gcloud artifacts repositories create $REPO_NAME --repository-format=docker \
```

Comment:
have you made sure that this command is a `CREATE IF NOT EXISTS` type?
will it run into issue if the repo already exists in the artifact registry?

Reply:
addressed in next commit, add a check for `CREATE IF NOT EXIST` type of command